python/python/lance/indices.py

```diff
@@ -396,6 +398,9 @@ def transform_vectors(
         dest_uri: str
             The URI to save the transformed vectors to.  The URI can be a local file
             path or a cloud storage path.
+        fragments: list[LanceFragment]
+            The list of data fragments to use when computing the transformed vectors.
+            Must be a non-empty list.
```

Comment:
Can we make it so that this is optional and, if not specified, all fragments will be used?